### DISCUSSION TITLE
Sqlitedb IGN France fail

### DISCUSSION THREAD

**Discussion by Louis-LJ** [osmandapp/OsmAnd#12642] [Q&A]
Hello, 

I followed the informartion to add an online map source of osmand. 

https://anygis.ru/Web/Html/Download_en?app=OsmandSqlite

I downloaded the sqlitedb IGN.fr Topo file, I put the map underlay but the map does not appear on osmand. 

Do you know why it doesn't work on android? with iphone it works very well.

**Comment by capta** [comment]
I tested and it doesn't work for me too on Android 9.
I'm interested if you find a solution.

**Reply by Louis-LJ** [reply]
IGN maps 

https://www.dropbox.com/s/cvv37ics9kvrlzc/IGN%20Maps.sqlitedb?dl=0

IGN Scan25

https://www.dropbox.com/s/wrwnsa9ljk5sbp1/IGN%20Scan25.sqlitedb?dl=0

**Reply by capta** [reply]
Thanks a lot it works great !

**Comment by bellera** [comment]
You can add IGN France topo as online source:

https://data.geopf.fr/private/wmts?apikey=ign_scan_ws&layer=GEOGRAPHICALGRIDSYSTEMS.MAPS&style=normal&tilematrixset=PM&Service=WMTS&Request=GetTile&Version=1.0.0&Format=image/jpeg&TileMatrix={0}&TileCol={1}&TileRow={2}

and use the cache for offline needs.

**Reply by EugeneZmeuk** [reply]
IGN France topo you can find on OsmAnd online maps list 
https://osmand.net/docs/user/map/raster-maps#select-raster-maps
